docs/guides/liquidations.md

```diff
@@ -0,0 +1,84 @@
+# Liquidation Guide
+
+This guide instructs users how to process a liquidation of an under-collateralized loan using polkadot.js.
+
+At the end of this guide you will know how to:
+
+- [x] [Funded your wallet with the debt token and gas token](#fund-wallet)
+- [x] [Liquidated a borrow position using polkadot.js](#liquidate-borrow-position)
+- [x] [Withdraw the received collateral tokens](#withdraw-collateral-optional)
+- [x] [Batch liquidations](#batch-liquidations)
+
+## Steps
+
+### **Fund wallet**
+
+1. The wallet needs as small amount of KINT tokens to pay for the gas fees.
+2. The wallet needs to be funded with the amount of `debt tokens` which shall be repaid. E.g. if the liquidator aims to liquidate a debt position of 1,000 USDT, the wallet needs to have at least this amount in the respective debt currency, in that case, USDT.
+
+All tokens that can be used in the money market can also be traded on Kintsugi DEX. For alternative options see [Other DEXs](#other-dexs)
+
+### **Liquidate borrow position**
+
+1. Go to [https://polkadot.js.org/apps/#/extrinsics](https://polkadot.js.org/apps/#/extrinsics) and select Kitnsugi network
+2. Set up the parameters for the call
+    1. **using the selected account**: `your_accound_address` (Kintsugi address, starting with ‘a3…’)
+    2. **submit the following extrinsic**: `loans.liquidateBorrow()`
+    3. **borrower:** `target_account_address` (Kintsugi address, starting with ‘a3…’)
+    4. **liquidationAssetId:** Currency which needs to be repaid
+        1. Select `Token` for KINT, kBTC, KSM
+        2. Select `ForeignAsset` for other tokens
+            1. see [here](#asset-registry) for `id:token` mapping
+    5. **repayAmount:** `amount_to_be_repaid` , note that the amount depends on the number of decimals the token uses. [See](#asset-registry) here for details.
+    6. **collateralAssetId:** The underlying of the currency to be received. E.g: For kBTC the liquidator will receive the lend token of kBTC which can be redeemed for kBTC (see [withdraw collateral](#withdraw-collateral-optional))
```

Comment:
```suggestion
    6. **collateralAssetId:** The currency to receive in exchange for liquidating the borrower's loan. The liquidation premium is also paid in this currency. Note that this currency has to be one of the collateral currencies used by the liquidated borrower. Also note that while `collateralAssetId` represents the underlying currency of a lending market (e.g. KBTC), the liquidator receives its qToken version instead (e.g. QKBTC), which can be redeem for KBTC from the lending market (see [withdraw collateral](#withdraw-collateral-optional)).
```

Reply:
I think we should use qToken terminology in general since the docs are already user facing, and hopefully hide the `LendToken` naming which is just an implementation detail. What do you think?

Reply:
Makes sense.